src/APL/vercel-apl.test.ts

```diff
@@ -56,6 +85,26 @@ describe("APL", () => {
       expect(apl["registerAppURL"]).toBe("option");
     });
 
+    describe("set", () => {
+      it("Successful save of the auth data", async () => {
+        const apl = new VercelAPL({
+          registerAppURL: "https://registerService.example.com",
+          deploymentToken: "token",
+        });
+        expect(await apl.set({ domain: "example.com", token: "token" })).toBe(undefined);
```

Comment:
This test doesnt really check anything. It doesnt check if auth data was saved - it checks if promise returns undefined.

You can check if request was delivered to msw in proper format. Or rename test to something like "doesn't throw for XYZ input" - but then this test is not too helpful